certificate-manager/src/test/java/io/strimzi/certs/OpenSslCertManagerTest.java

```diff
@@ -116,6 +118,8 @@ private void testGenerateSelfSignedCert(File key, File cert, Subject sbj) throws
     public void testGenerateSignedCert() throws Exception {
 
         File caKey = File.createTempFile("ca-key-", ".key");
+        Path path = caKey.toPath().getParent();
+        long fileCount = Files.list(path).count() - 1;
```

Comment:
Why the `- 1`?

Reply:
We want to know how many files is in the folder. We create `caKey` file in temp folder so we will increase this count by 1. Then we get the path, create all other files, ... I suppose it would be better to remove this assert (and count) or we should get the tmp path another way. For example hardcoded `"/tmp"`.